README.md

```diff
@@ -1334,17 +1333,16 @@ async function startRhino() {
     RHINO_MODEL_BASE64
   );
 
-  // Start up the web voice processor. It will request microphone permission
+  // Initialize the web voice processor.
   // It downsamples the audio to voice recognition standard format (16-bit 16kHz linear PCM, single-channel)
-  // The incoming microphone audio frames will then be forwarded to the Rhino Worker
-  // n.b. This promise will reject if the user refuses permission! Make sure you handle that possibility.
-  const webVp = await WebVoiceProcessor.init({
-    engines: [rhino.worker],
-    start: true,
-  });
+  // The incoming microphone audio frames will then be forwarded to the Rhino Worker.
+  webVp = await WebVoiceProcessor.WebVoiceProcessor.instance();
+  webVp.subscribe(rhino.worker);
```

Comment:
with the new interface, you just pass `rhino` and it will determine whether it's a worker or not

Reply:
Right, good catch. It works either way I guess.